buildlib/pr/tests.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           ./contrib/test_jenkins.sh
           exit $?
         displayName: Run ./contrib/test_jenkins.sh
+        retryCountOnTaskFailure: 2
```

Comment:
which tests would it affect?

Reply:
This change should eventually affect every test that might fail due to network timeout etc.
Let's see how it works and evaluate the need to extend it to IO demo tests too.

Reply:
This may hide random, but real, issues 
I prefer to review failure before restarting.
we can add it only to build tests, which are deterministic.

Reply:
Good point!
For now I'm using this PR to test `/azp run perf` failures.
Please, unsubscribe.